config.py

```diff
@@ -0,0 +1,73 @@
+"""Configuration module for dof-chat application."""
+
+from pydantic import field_validator
+try:
+    from pydantic_settings import BaseSettings
+except ImportError:
+    # Fallback for older pydantic versions or missing pydantic-settings
+    from pydantic import BaseModel as BaseSettings
+
+class Settings(BaseSettings):
+    """Application settings using Pydantic Settings."""
+    
+    # Database configuration
+    database_path: str = "dof_db/db.duckdb"
+    
+    # Gemini API configuration
+    # TODO: Enable API key validation for production deployment
```

Comment:
Again, I don't like TODOs, I would remove this code and instead create a ticket.